src/lib/processOpenAPI/processOpenAPI.ts

```
@@ -1,11 +1,12 @@
 import type { OpenAPI, OpenAPIV3 } from '@scalar/openapi-types'
 import type { JSONSchema } from '@trojs/openapi-dereference'
-import type { ParsedContent, ParsedOpenAPI, ParsedOperation } from '../types'
+import type { ParsedContent, ParsedOpenAPI, ParsedOperation } from '../../types'
 import { dereferenceSync } from '@trojs/openapi-dereference'
 import { merge } from 'allof-merge'
-import { getSchemaExample } from './examples/getSchemaExample'
+import { getSchemaExample } from '../examples/getSchemaExample'
 import { getSchemaUi } from './getSchemaUi'
 import { getSecurityUi } from './getSecurityUi'
+import { getCodeSamples } from './getCodeSamples'
```

ESLint

[error] Expected "./getCodeSamples" to come before "./getSecurityUi".

(perfectionist/sort-imports)